src/datascience-ui/error-renderer/index.ts

```diff
@@ -96,6 +97,11 @@ export const activate: ActivationFunction = (_context) => {
                         line = line.replace(matches[0], `<a href='command:${matches[1]}'>${matches[2]}</a>`);
                     }
                 }
+                while ((matches = akaMsLinks.exec(line)) !== null) {
```

Comment:
Modified to support https links to aka.ms links
This way the `More info` is also displayed in the error output